tests/Unit/EnforcerTest.php

```diff
@@ -241,9 +241,9 @@ public function testGetImplicitPermissionsForUser()
 
     public function testGetImplicitUsersForPermission()
     {
-        $e = new Enforcer($this->modelAndPolicyPath . '/rbac_model.conf', $this->modelAndPolicyPath . '/rbac_with_hierarchy_policy.csv');
+        $e = new Enforcer($this->modelAndPolicyPath . '/rbac_model.conf', $this->modelAndPolicyPath . '/rbac_policy.csv');
+
         $this->assertEquals($e->getImplicitUsersForPermission('data1', 'read'), ['alice']);
-        $this->assertEquals($e->getImplicitUsersForPermission('data1', 'write'), ['alice']);
```

Comment:
Why ?

Reply:
```php
   $this->assertEquals($e->getImplicitUsersForPermission('data1', 'read'), ['alice']);
   $this->assertEquals($e->getImplicitUsersForPermission('data1', 'write'), ['alice']);

```
I think these two pieces of code have the same effect. Although they used different variables